hw/sensor/pkg.yml

```diff
@@ -23,6 +23,12 @@ pkg.author: "Apache Mynewt <dev@mynewt.incubator.apache.org>"
 pkg.homepage: "http://mynewt.apache.org/"
 pkg.keywords:
 
+pkg.deps:
+    - sys/shell
+    - sys/console/full
+    - sys/log/full
+    - sys/stats/full
```

Comment:
This was a remnant from the earlier implementation where the sensor library itself used logs for the oic implementation. It doesn't use that anymore so, it can be improved. However, I think keeping shell and console makes sense as they should be depended on by the sensor package. So, I am thinking of doing this instead:

pkg.deps.SENSOR_CLI:
   - sys/shell
   - sys/console/full